presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -311,16 +307,14 @@ private JdbcOutputTableHandle beginWriteTable(ConnectorTableMetadata tableMetada
                 }
                 columnNames.add(columnName);
                 columnTypes.add(column.getType());
-                columnList.add(new StringBuilder()
-                        .append(quoted(columnName))
-                        .append(" ")
-                        .append(toSqlType(column.getType()))
-                        .toString());
+                columnList.add(format("%s %s", quoted(columnName), toSqlType(column.getType())));
```

Comment:
is collection`columnList` needed? We could reuse `columnNames` and `columnTypes`.